ARExperiment/README.md

```diff
@@ -0,0 +1,32 @@
+# ARExperiment
+
+This is a simple demo app that pairs with our investigation into ARKIT
+
+* `OneModelUsingAnchorsViewController` Adds one 3D model to the scene using `ARAnchors` method described on the [Getting started with ARKit](https://www.novoda.com/blog/getting-started-with-arkit/) blog post
+* `OneModelUsingVectorsViewController` Adds one 3D model to the scene using `SCNNodes` and `planeDetection` method described on the [Getting started with ARKit](https://www.novoda.com/blog/getting-started-with-arkit/) blog post
+* `SizeComparisonViewController` Adds 3 different sized cubes to the `sceneView` to check how big cm are in the AR world
```

Comment:
To check or to demonstrate?

Reply:
don't quite get this one

Reply:
@dkaravias demonstrate would be better thank you